src/classes/UTIL_RecordTypes_TEST.cls

```diff
@@ -40,47 +40,40 @@ private class UTIL_RecordTypes_TEST {
     // TEST
     static testmethod void testRecTypes() {
         // try bogus values
-        Id reallyBogus = UTIL_RecordTypes.GetRecordTypeId('Bogus', 'Bogus');
-        Id bogus = UTIL_RecordTypes.GetRecordTypeId('Opportunity', 'Bogus');
-        bogus = UTIL_RecordTypes.GetRecordTypeId('Contact', 'Bogus');
+        Id bogus = UTIL_RecordTypes.GetRecordTypeId(Opportunity.sObjectType, 'Bogus');
+        bogus = UTIL_RecordTypes.GetRecordTypeId(Contact.sObjectType, 'Bogus');
         bogus = UTIL_RecordTypes_API.GetRecordTypeId('Campaign', 'Bogus');
         
         // try all the functions
         set<Id> oppRtSet = UTIL_RecordTypes_API.GetRecordTypeIdSet('Opportunity', (new set<string>{'Bogus', 'Master', 'Grant'}));
-        Map<String, Id> oppTypeMap = UTIL_RecordTypes.GetRecordTypeIdMap('Opportunity');
-        List<SelectOption> rtOptions = UTIL_RecordTypes.getRecordTypesForSelectList('Opportunity');
+        List<SelectOption> rtOptions = UTIL_RecordTypes.getRecordTypesForSelectList(Opportunity.sObjectType);
         
         // check against queried rectypes
         list<recordtype> ort = [select id, name from recordtype where isactive=true and sobjecttype='Opportunity' limit 1];
         if (!ort.isEmpty()) {
-            Id rtId = UTIL_RecordTypes.GetRecordTypeId('Opportunity', ort[0].name);
+            Id rtId = UTIL_RecordTypes.GetRecordTypeId(Opportunity.sObjectType, ort[0].name);
             system.assertEquals(ort[0].id, rtId); 
             String rtName = UTIL_RecordTypes_API.GetRecordTypeName('Opportunity', ort[0].id);
             system.assertEquals(ort[0].name, rtName); 
 
             // the sets above should contain something too
             system.assert(oppRtSet.size() > 0);
-            system.assert(oppTypeMap.size() > 0);
             system.assert(rtOptions.size() > 0);            
         }       
     }
     
     static testMethod void TestRecordTypes() {
 
-        ID idGift = UTIL_RecordTypes.GetRecordTypeId('Opportunity', UTIL_RecordTypes.getRecordTypeNameForGiftsTests('Opportunity'));
+        ID idGift = UTIL_RecordTypes.GetRecordTypeId(Opportunity.sObjectType, UTIL_RecordTypes.getRecordTypeNameForGiftsTests(Opportunity.sObjectType));
         
-        Set<String> setStr = new Set<String>{UTIL_RecordTypes.getRecordTypeNameForGiftsTests('Opportunity')};
-        Set<Id> setId = UTIL_RecordTypes.GetRecordTypeIdSet('Opportunity', setStr);
+        Set<String> setStr = new Set<String>{UTIL_RecordTypes.getRecordTypeNameForGiftsTests(Opportunity.sObjectType)};
+        Set<Id> setId = UTIL_RecordTypes.GetRecordTypeIdSet(Opportunity.sObjectType, setStr);
         if(idGift!=null){
             system.Assert(setId.contains(idGift));
         
-            Map<String, Id> mapRCId = UTIL_RecordTypes.GetRecordTypeIdMap('Opportunity');
```

Comment:
I dropped this line because I removed the `GetRecordTypeIdMap` method, since it was not being used by any external code